### PR TITLE
Add specifications according to the Pypi

### PR DESCRIPTION
**Pull request by imhassantariq**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Description
+###########
+
+Caliper Tracking can be used to transform the edx traditional event logs into `IMS Caliper Standard <http://imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html>`_ format. Then those logs can be used with any analytics application that is compatible with Caliper Standard events.
+
+Installation
+############
+
+To install **caliper-tracking** in Openedx add the following inside your
```
Comment (asamolion) [inline README.rst]: To install **caliper-tracking** in your Open edX installation add the following inside your requirements file. (For most Open edX installations it is located at ``edx-platform/requirements/edx/base.txt``::

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    caliper-tracking==0.9.0
+
+
+Usage
+#####
+
+To enable and use `caliper-tracking`
+
+Add/Enable the `ENABLE_EVENT_CALIPERIZATION` flag in `FEATURES` in the following files:
```
Comment (asamolion) [inline README.rst]: Set the value of `ENABLE_EVENT_CALIPERIZATION` flag

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+
+    "FEATURES": {
+        ...
+
+        "ENABLE_EVENT_CALIPERIZATION": true,
+
+    }
+
+
```
Comment (asamolion) [inline README.rst:40]: Add a heading like `Location of output logs`

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+Please see `LICENSE <./LICENSE>`_ for details.
+
+
+How To Contribute
+#################
+
+To contribute make PR in this repositry on Github : `Caliper Tracking <https://github.com/ucsd-ets/caliper-tracking>`_
+
+If the problem persists, feel free to open an issue in our Github Repository: `Caliper Tracking <https://github.com/ucsd-ets/caliper-tracking>`_
```
Comment (asamolion) [inline README.rst]: For issues and questions, feel free

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+
+Contributors
```
Comment (asamolion) [inline README.rst:64]: Ask @zee-pk and add the names of UCSD's key people as well.

**Review by asamolion — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (imhassantariq) [reply README.rst]: Done

File: README.rst (hunk 2 above)
Comment (imhassantariq) [reply README.rst]: Done

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+    "FEATURES": {
+        ...
+
+        "ENABLE_EVENT_CALIPERIZATION": true,
+
+    }
+
+
+Transformed logs location
```
Comment (asamolion) [inline README.rst]: The sentence isn't grammatically correct. Use `Location of`

**Review by asamolion — CHANGES_REQUESTED** [review]

**Review by asamolion — APPROVED** [review]

**Pull request merged**